docs/zh_CN/CommunitySharings/NniPracticeSharing/HpoComparison.md

```diff
@@ -71,33 +71,33 @@
 
```

Comment:
I have a PR to change path of this file. Hope to help review:
https://github.com/microsoft/nni/pull/1097

Reply:
You may need to change path, too.

Reply:
fixed